### PR TITLE
GitHub actions setup

### PR DESCRIPTION
**Pull request by niyatim23**
*Issue #, if available:*

*Description of changes:*
All builds working on GitHub Actions except Windows and builds that are allowed to fail on travis (MSAN, TSAN, UBSAN)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,81 +29,34 @@ matrix:
...
-      before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DADDRESS_SANITIZER=TRUE -DBUILD_GSTREAMER_PLUGIN=TRUE -DBUILD_JNI=TRUE
 
     # UndefinedBehaviorSanitizer
-    - name: "Linux Clang UndefinedBehaviorSanitizer"
-      os: linux
-      compiler: clang
-      env: allowTestFail=true
-      # env: UBSAN_OPTIONS=halt_on_error=1
-      before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DUNDEFINED_BEHAVIOR_SANITIZER=TRUE -DBUILD_GSTREAMER_PLUGIN=TRUE -DBUILD_JNI=TRUE
+    # - name: "Linux Clang UndefinedBehaviorSanitizer"
```
Comment (disa6302) [inline .travis.yml:34]: Let us uncomment all the sanitizers on travis.

**Review by disa6302 — COMMENTED** [review]
Include a mac-os latest and ubuntu latest build as well.

File: .travis.yml (hunk 1 above)
Comment (niyatim23) [reply .travis.yml]: Uncommented in the latest commit

**Comment by disa6302** [comment]
Remove ubsan from travis file. Given we only have allowedToFail builds in Travis, I think we can add those jobs commented in the github actions file and remove the travis file altogether.

**Review by disa6302 — APPROVED** [review]

**Pull request merged**